renderer/render-markdown.js

```diff
@@ -116,7 +116,7 @@ function gemojiToImages() {
 }
 
 function fixHeadings() {
```

Comment:
I think the whole `fixHeadings` handler can be removed. It looks like all it does is make it possible to create headings without spaces after the `#`.